services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_thin.py

```diff
@@ -209,6 +214,16 @@ async def post_containers_tasks_state_save(
         url = self._get_url(dynamic_sidecar_endpoint, "/containers/state:save")
         return await self.client.post(url)
 
+    @retry_on_errors()
+    @expect_status(status.HTTP_202_ACCEPTED)
+    async def post_containers_user_services_images_pull(
+        self, dynamic_sidecar_endpoint: AnyHttpUrl
+    ) -> Response:
+        url = self._get_url(
+            dynamic_sidecar_endpoint, "/containers/user-services/images:pull"
```

Comment:
question: why not just "/container/images:pull" ?

Reply:
yes that's the better one, since containers and user-services are synonyms here